.github/workflows/ci.yml

```diff
@@ -13,5 +13,10 @@ on:
 
 jobs:
   call-terraform-ci-pipeline:
-    uses: terraform-ibm-modules/common-pipeline-assets/.github/workflows/common-terraform-module-ci.yml@v1.8.1
+    uses: terraform-ibm-modules/common-pipeline-assets/.github/workflows/common-terraform-module-ci.yml@v1.9.1
     secrets: inherit
+    with:
+      craSCCv2: true
+      craTarget: "examples/ibm-catalog/deployable-architectures/full-stack"
+      #craGoalIgnoreFile: "cra-tf-validate-ignore-goals.json"
+      craEnvironmentVariables: "TF_VAR_prefix=full-infra,TF_VAR_powervs_zone=tok04,TF_VAR_powervs_resource_group_name=Default,TF_VAR_ssh_private_key=${{ secrets.CRA_SSH_PRIVATE_KEY }},TF_VAR_preset=${{ secrets.CRA_PRESET_EOF }},TF_VAR_ssh_public_key=${{ secrets.CRA_SSH_PUBLIC_KEY }}"
```

Comment:
OK, so this fails with `The workflow is not valid. .github/workflows/ci.yml (Line: 22, Col: 32): Unrecognized named-value: 'secrets'. Located at position 1 within expression: secrets.CRA_SSH_PRIVATE_KEY`

Might need to pull in @daniel-butler-irl here...

Reply:
@surajsbharadwaj As a workaround, how about using variable for all here, since we wont be adding any real actual valid secrets?

Reply:
@ocofaigh you cannot use vars as it doesnt support whitespaces :(

Reply:
So thats an issue for the pub and prv keys? Is validation done on terraform plan for those, or only on apply? I dont see any variable validation, so perhaps we can entter any value?